fulfillments.go

```diff
@@ -60,36 +59,33 @@ func (repository fulfillmentRepository) Create(orderID int64, fulfillment shopif
 	return response.Fulfillment.ToShopify(), nil
 }
 
-func (repository fulfillmentRepository) Update(orderID int64, fulfillmentID int64, update shopify.Fulfillment) (shopify.Fulfillment, error) {
-	updateDTO := FulfillmentDTO{
-		ID:              fulfillmentID,
-		TrackingCompany: update.TrackingCompany,
-		TrackingNumbers: update.TrackingNumbers,
-		TrackingURLs:    update.TrackingURLs,
-		NotifyCustomer:  update.NotifyCustomer,
+func (repository fulfillmentRepository) UpdateTracking(update shopify.Fulfillment) (shopify.Fulfillment, error) {
+	updateTrackingDTO := FulfillmentDTO{
+		TrackingInfo:   BuildTrackingInfoDTO(update),
+		NotifyCustomer: update.NotifyCustomer,
 	}
 
 	request := struct {
 		Fulfillment FulfillmentDTO `json:"fulfillment"`
 	}{
-		Fulfillment: updateDTO,
+		Fulfillment: updateTrackingDTO,
 	}
 
 	body, err := json.Marshal(request)
 	if err != nil {
 		return shopify.Fulfillment{}, err
 	}
 
-	url := repository.createURL(fmt.Sprintf("orders/%v/fulfillments/%v.json", orderID, fulfillmentID))
+	url := repository.createURL(fmt.Sprintf("fulfillments/%v/update_tracking.json", update.ID))
```

Comment:
Any reason to use %v instead of %d here?

Reply:
No reason! Just me amending the code in place, will add :smile: